rules/prefer-add-event-listener.js

```diff
@@ -2,6 +2,8 @@
 const getDocsUrl = require('./utils/get-docs-url');
 const domEventsJson = require('./utils/dom-events.json');
 
+const excludedPackages = new Set(['koa', 'sax']);
```

Comment:
Should we add an option to override or extend this set?

Reply:
Yeah, we could add a configuration option, but I think we should keep these values as a default if no options are specified.